### ISSUE TITLE
Bug:  The wrapper methods props can't get specific prop in functional component

### ISSUE THREAD

**Issue by edisonLzy** [vuejs/test-utils#1311]
**Describe the bug**

The wrapper methods props can't get specific prop in functional component
**To Reproduce**

const wrapper = mount(Component,{
      props: {
        title: 'title'
      }
    });
 expect(wrapper.props('title')).toContain('title')  //  not pass  , the wrapper.props('title') is undeinfed

**Expected behavior**

    const wrapper = mount(Component,{
      props: {
        title: 'title'
      }
    });
    expect(wrapper.props('title')).toContain('title')  // pass

**Related information:**
- `@vue/test-utils` version: ^2.0.0-rc.17
- `Vue` version: 3.2.27
- `node` version: v16.0.0
- `npm` (or `yarn`) version: v8.3.0

**Additional context**

**Comment by cexbrayat** [comment]
Hmm, interesting. I guess not a lot of people are using functional components in Vue 3 and we never caught this. Would you like to try to fix it? 

You can start by adding a test in `props.spec.ts` https://github.com/vuejs/test-utils/blob/main/tests/props.spec.ts to reproduce your error.

Then the relevant code is here:
https://github.com/vuejs/test-utils/blob/67a3e7a937a6392ca6b092f97bf08f00f17360d3/src/vueWrapper.ts#L126-L131

We probably need some special handling for functional components.

**Comment by edisonLzy** [comment]
woot 😄 .  i find the problem is the functional component wrapper ignored  the props options.
The code is here: 
https://github.com/vuejs/test-utils/blob/67a3e7a937a6392ca6b092f97bf08f00f17360d3/src/mount.ts#L281-L301

I have an idea is add instance methods name `attrs` in wrapper like this:

```js
const wrapper = mount(xx);
wrapper.attrs('title') === title
```

**Comment by cexbrayat** [comment]
Fixed by #1326
